lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -296,3 +298,10 @@ const getSelected = (population: Population, selectedPopulations: Population[]):
   });
   return selected;
 };
+
+const getCmsId = (measureResourceJson: string): string => {
+  const measureResource = JSON.parse(measureResourceJson);
+  const identifier = measureResource.identifier;
```

Comment:
slight ambiguous. can we rename this to `identifiers` and then `identifiers.find(..)` or just inline `measureResource.identifier?.find`

Reply:
Just did thanks!

Reply:
@adongare I seem to have vulnerability problems: 
source-map-resolve  >=0.5.1
  Depends on vulnerable versions of decode-uri-component, 
can you please help take a look? Thanks!

Reply:
doesn't look like it has a patch available. @sb-benohe can we ignore it for now? this is dev/test dependency. we'll find alternative to this.

Reply:
nvm, we are good with it now. removed vulnerable package. Thanks, Cecilia.